tests/test-definition.yaml

```diff
@@ -0,0 +1,39 @@
+---
+dimensions:
+  - name: druid
+    values:
+      - 0.22.1-stackable0.1.0
```

Comment:
`0.22.1-stackable0` (but not `0.1.0`) resolved to the image including the authorizer, so we need the `0.2.0` here :
```suggestion
      - 0.22.1-authorizer0.1.0-stackable0.2.0
```

Reply:
We should probably fix this in the docker-images repo instead.

Reply:
Agreed - I'll open a separate issue for this.